moto/iam/responses.py

```diff
@@ -87,6 +87,12 @@ def list_instance_profiles(self):
         template = self.response_template(LIST_INSTANCE_PROFILES_TEMPLATE)
         return template.render(instance_profiles=profiles)
 
+    def list_instance_profiles_for_role(self, role_name=None):
+        profiles = iam_backend.get_instance_profiles_for_role(role_name=role_name)
```

Comment:
Ahhh ok.  Will make the change and undo the change to `responses.py`.

Thank you!